paasta_tools/frameworks/adhoc_scheduler.py

```diff
@@ -60,3 +61,7 @@ def tasks_and_state_for_offer(self, driver, offer, state):
 
     def kill_tasks_if_necessary(self, *args, **kwargs):
         return
+
+    def validate_config(self):
```

Comment:
"validate_config" implies that we are validating the entire config. Since we are only checking if cmd exists or not, can we rename this to something like "check_if_cmd_exists"?